.github/workflows/ci.yml

```
@@ -0,0 +1,64 @@
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements-strict.txt
+          pip install ruff
+      - name: Ruff lint
+        run: ruff check .
+      - name: Run tests
+        run: pytest
+      - name: Backtest pipeline
+        run: |
+          python -c '
+from src import data_prep, strategy
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 27: could not find expected ':'

(syntax-check)